### PR TITLE
Correct Timestamps / TAI text

### PR DESCRIPTION
**Pull request by peterbrightwell**
Timestamps should use SMPTE epoch. Better links for leap second correction.

Most of the below commits were to do with the reordering so will need squashing.

### CODE REVIEW

File: docs/4.0. Behaviour.md [hunk 1]
```diff
@@ -248,11 +248,9 @@ A client cancels an activation by submitting a DELETE request to the resource `/
 
 ### Timestamps
 
-All timestamps in this API MUST be string formatted timestamps (\<seconds\>:\<nanoseconds\>) indicating time (absolute or relative) for activation. For relative times, the time '0:0' MUST be considered to be the time the request is received by the API.
+Timestamps indicate the absolute or relative time for activation and MUST be string formatted as (\<seconds\>:\<nanoseconds\>). Absolute timestamps MUST be based on TAI (which does not apply leap seconds) and must use SMPTE epoch (meaning 1970-01-01 00:00:00 TAI would be represented as '0:0').
```
Comment (garethsb) [inline docs/4.0. Behaviour.md]:
```suggestion
Timestamps indicate the absolute or relative time for activation and MUST be string formatted as \<seconds\>:\<nanoseconds\>. Absolute timestamps MUST be based on TAI (which does not apply leap seconds) and must use SMPTE epoch (meaning 1970-01-01 00:00:00 TAI would be represented as '0:0').
```

Comment (peterbrightwell) [reply docs/4.0. Behaviour.md]: 52c86ad73d670874a4ee94140d7d0655258964c7 formats that as code

**Review by garethsb — APPROVED** [review]
LGTM, and should merge nicely so long as done after PR #40.

**Review by chris-gil — APPROVED** [review]
Additional link to list of time offsets completes this work.

**Pull request merged**